gateway/1.4/modules/ROOT/pages/flex-view-replica-status.adoc

```diff
@@ -5,21 +5,45 @@ endif::[]
 :page-aliases: runtime-manager::flex-gateway-replica-status.adoc
 :imagesdir: ../assets/images
 
-To view information about the status of Flex Replicas:
+In Runtime Manager, you can view the status of groups of Flex Gateway replicas and individual replicas.
```

Comment:
```suggestion
In Runtime Manager, you can view the status of groups of Flex Replicas and individual Flex Replicas.
```
It was actually decided by PM that we call these "Flex Replicas". Please use this throughout. Otherwise we'll have to change it everywhere for consistency.